commit bba300da5347d8ac25544650c6f6e507e5ea8a1b
Author: Gwenole Beauchesne <gbeauchesne@splitted-desktop.com>
Date:   Thu Sep 24 12:00:02 2009 +0000

    [G45] Fix vaQueryImageFormats() to return 0 image formats supported at this time.

diff --git a/i965_drv_video/i965_drv_video.c b/i965_drv_video/i965_drv_video.c
index 375d532..8d28547 100644
--- a/i965_drv_video/i965_drv_video.c
+++ b/i965_drv_video/i965_drv_video.c
@@ -346,6 +346,9 @@ i965_QueryImageFormats(VADriverContextP ctx,
                        VAImageFormat *format_list,      /* out */
                        int *num_formats)                /* out */
 {
+    if (num_formats)
+        *num_formats = 0;
+
     return VA_STATUS_SUCCESS;
 }
 
